__tests__/app.test.js

```diff
@@ -123,10 +123,15 @@ describe("/api/articles/:article_id", () => {
 				});
 		});
 	});
-	describe("GET: 204", () => {
-		it("should return 204 if article is not in DB", () => {
+	describe("GET: 404", () => {
```

Comment:
please make sure you only make code amendments that are relevant to the ticket your working on, it creates a confusing git history and can make your reviewers life more difficult than it need to be 🤣